InMobi/src/main/java/com/mopub/mobileads/InMobiAdapterConfiguration.kt

```diff
@@ -86,11 +86,12 @@ class InMobiAdapterConfiguration : BaseAdapterConfiguration() {
                 val accountId = getAccountId(configuration)
                 InMobiSdk.init(context, accountId, null) {
                     if (it == null) {
-                        initCompletionListener.onSuccess()
+                        MoPubLog.log(CUSTOM, ADAPTER_NAME, "InMobi initialization success.")
                     } else {
-                        initCompletionListener.onFailure(it, null)
```

Comment:
Any reason to remove the failure callback?

Reply:
https://github.com/mopub/mopub-android-mediation/pull/311#discussion_r573167988

check this